--- a/src/Makefile.am	2016-03-18 00:32:29.250078300 +0900
+++ b/src/Makefile.am	2016-03-18 00:32:24.390618500 +0900
@@ -5,7 +5,7 @@ EXTSOURCES =	xproto.c \
 		xc_misc.c
 
 AM_CFLAGS = $(BASE_CFLAGS) $(NEEDED_CFLAGS) $(XDMCP_CFLAGS)
-libxcb_la_LIBADD = $(NEEDED_LIBS) $(XDMCP_LIBS)
+libxcb_la_LIBADD = $(NEEDED_LIBS) $(XDMCP_LIBS) -lws_32
 libxcb_la_SOURCES = \
 		xcb_conn.c xcb_out.c xcb_in.c xcb_ext.c xcb_xid.c \
 		xcb_list.c xcb_util.c xcb_auth.c c_client.py
