diff --git a/src/dosinst.c b/src/dosinst.c
index 05398be..436a4ff 100644
--- a/src/dosinst.c
+++ b/src/dosinst.c
@@ -2117,7 +2117,7 @@ install_vimfilesdir(int idx)
 
     /* Now, just create the directory.	If it already exists, it will fail
      * silently.  */
-    sprintf(vimfiles_path, "%s\\vimfiles", vimdir_path);
+    sprintf(vimfiles_path, "%s\\.vim", vimdir_path);
     vim_mkdir(vimfiles_path, 0755);
 
     printf("Creating the following directories in \"%s\":\n", vimfiles_path);
@@ -2150,7 +2150,7 @@ init_directories_choice(void)
     if (getenv("HOME") != NULL)
     {
 	vimfiles_dir_choice = (int)vimfiles_dir_home;
-	sprintf(tmp_dirname, "%s\\vimfiles\\compiler", getenv("HOME"));
+	sprintf(tmp_dirname, "%s\\.vim\\compiler", getenv("HOME"));
 	if (stat(tmp_dirname, &st) == 0)
 	    vimfiles_dir_choice = (int)vimfiles_dir_none;
     }
@@ -2162,7 +2162,7 @@ init_directories_choice(void)
 	    dir_remove_last(installdir, tmp_dirname);
 	else
 	    strcpy(tmp_dirname, p);
-	strcat(tmp_dirname, "\\vimfiles\\compiler");
+	strcat(tmp_dirname, "\\.vim\\compiler");
 	if (stat(tmp_dirname, &st) == 0)
 	    vimfiles_dir_choice = (int)vimfiles_dir_none;
     }
diff --git a/src/os_dos.h b/src/os_dos.h
index 0edc3e0..fc021e9 100644
--- a/src/os_dos.h
+++ b/src/os_dos.h
@@ -17,13 +17,13 @@
 # define SYS_VIMRC_FILE		"$VIM\\vimrc"
 #endif
 #ifndef USR_VIMRC_FILE
-# define USR_VIMRC_FILE		"$HOME\\_vimrc"
+# define USR_VIMRC_FILE		"$HOME\\.vimrc"
 #endif
 #ifndef USR_VIMRC_FILE2
-# define USR_VIMRC_FILE2	"$HOME\\vimfiles\\vimrc"
+# define USR_VIMRC_FILE2	"$HOME\\.vim\\vimrc"
 #endif
 #ifndef USR_VIMRC_FILE3
-# define USR_VIMRC_FILE3	"$VIM\\_vimrc"
+# define USR_VIMRC_FILE3	"$VIM\\.vimrc"
 #endif
 #ifndef VIM_DEFAULTS_FILE
 # define VIM_DEFAULTS_FILE	"$VIMRUNTIME\\defaults.vim"
@@ -44,13 +44,13 @@
 #  define SYS_GVIMRC_FILE	"$VIM\\gvimrc"
 # endif
 # ifndef USR_GVIMRC_FILE
-#  define USR_GVIMRC_FILE	"$HOME\\_gvimrc"
+#  define USR_GVIMRC_FILE	"$HOME\\.gvimrc"
 # endif
 # ifndef USR_GVIMRC_FILE2
-#  define USR_GVIMRC_FILE2	"$HOME\\vimfiles\\gvimrc"
+#  define USR_GVIMRC_FILE2	"$HOME\\.vim\\gvimrc"
 # endif
 # ifndef USR_GVIMRC_FILE3
-#  define USR_GVIMRC_FILE3	"$VIM\\_gvimrc"
+#  define USR_GVIMRC_FILE3	"$VIM\\.gvimrc"
 # endif
 # ifndef SYS_MENU_FILE
 #  define SYS_MENU_FILE		"$VIMRUNTIME\\menu.vim"
@@ -63,24 +63,24 @@
 
 #ifdef FEAT_VIMINFO
 # ifndef VIMINFO_FILE
-#  define VIMINFO_FILE		"$HOME\\_viminfo"
+#  define VIMINFO_FILE		"$HOME\\.viminfo"
 # endif
 # ifndef VIMINFO_FILE2
-#  define VIMINFO_FILE2		"$VIM\\_viminfo"
+#  define VIMINFO_FILE2		"$VIM\\.viminfo"
 # endif
 #endif
 
 #ifndef VIMRC_FILE
-# define VIMRC_FILE	"_vimrc"
+# define VIMRC_FILE	".vimrc"
 #endif
 
 #ifndef EXRC_FILE
-# define EXRC_FILE	"_exrc"
+# define EXRC_FILE	".exrc"
 #endif
 
 #ifdef FEAT_GUI
 # ifndef GVIMRC_FILE
-#  define GVIMRC_FILE	"_gvimrc"
+#  define GVIMRC_FILE	".gvimrc"
 # endif
 #endif
 
@@ -116,7 +116,7 @@
 #endif
 
 #ifndef DFLT_VDIR
-# define DFLT_VDIR	"$VIM/vimfiles/view"	/* default for 'viewdir' */
+# define DFLT_VDIR	"$VIM/.vim/view"	/* default for 'viewdir' */
 #endif
 
 #ifndef DFLT_DIR
@@ -124,7 +124,7 @@
 #endif
 
 #define DFLT_ERRORFILE		"errors.err"
-#define DFLT_RUNTIMEPATH	"$HOME/vimfiles,$VIM/vimfiles,$VIMRUNTIME,$VIM/vimfiles/after,$HOME/vimfiles/after"
+#define DFLT_RUNTIMEPATH	"$HOME/.vim,$VIM/.vim,$VIMRUNTIME,$VIM/.vim/after,$HOME/.vim/after"
 
 #define CASE_INSENSITIVE_FILENAME   /* ignore case when comparing file names */
 #define SPACE_IN_FILENAME
